.github/workflows/build.yml

```diff
@@ -184,14 +180,18 @@ jobs:
           ${{ env.ACE_ROOT }}/bin/mwc.pl -type gnuace -exclude ACE_TAO,OpenDDS
           make -j2 || make
           echo "Test ========== ========== =========="
-          if [ ${{ matrix.m.dds_security }} == 1 ]; then TEST_OPTIONS+="--secure"; fi
           cd test
-          ./run_test.pl cpp2node $TEST_OPTIONS
-          ./run_test.pl node2cpp $TEST_OPTIONS
-          ./run_test.pl node2node $TEST_OPTIONS
+          ./run_test.pl cpp2node
+          ./run_test.pl node2cpp
+          ./run_test.pl node2node
           ./run_test.pl cpp2node --rtps
           ./run_test.pl node2cpp --rtps
           ./run_test.pl node2node --rtps
+          if [ ${{ matrix.m.dds_security }} == 1 ]; then
+          ./run_test.pl cpp2node --rtps --secure;
```

Comment:
indent this block... also I'm not sure the `;` is needed since it's the end of the line